src/sed.js

```diff
@@ -25,6 +25,19 @@ common.register('sed', _sed, {
 //@
 //@ Reads an input string from `files` and performs a JavaScript `replace()` on the input
 //@ using the given search regex and replacement string or function. Returns the new string after replacement.
+//@
+//@ Note:
+//@
+//@ Unix `sed` specifies capture groups using `\n` syntax. In ShellJS (and JavaScript, in general),
```

Comment:
Don't use `\n`, since that sounds like the newline character. It might suffice to just provide the javascript example and skip the shell code, for brevity's sake.